tslint.json

```diff
@@ -1,5 +1,7 @@
 {
-    "extends": [
-        "@dcos/tslint-config"
-    ]
+  "extends": ["@dcos/tslint-config"],
+  "rules": {
+    "interface-name": [false],
+    "no-submodule-imports": [false]
```

Comment:
Can we have a small note to why `no-submodule-imports` on this PR description?

Reply:
> `Submodule import paths from this package are disallowed; import from the root instead`  when importing via `#ALIAS/`

https://mesosphere.slack.com/archives/C03UXUVH7/p1528733271000724

Reply:
Thanks.